### PR TITLE
Inject db reader writer

### PR DESCRIPTION
**Pull request by Plippe**
This is a way of injecting code when launching a lua script. This will allow us to obtain at least two outputs, one for dev, and one for prod.

### CODE REVIEW

File: src/torch/0-tiefvision-commons/tiefvision_config_loader.lua [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    end
+  end
+
+  return nil
+end
+
+local configLoader = {}
+
+configLoader.argument = argument(arg)
+configLoader.environment = os.getenv('CONFIG')
```
Comment (emersonloureiro) [inline src/torch/0-tiefvision-commons/tiefvision_config_loader.lua]: Sounds a bit weird that the `environment` is under `CONFIG` environment variable. Why not `ENVIRONMENT`?

File: src/torch/0-tiefvision-commons/tiefvision_config_loader.lua [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+configLoader.default = require('tiefvision_commons').path('src/torch/config.lua')
+
+configLoader.file = configLoader.argument or
+  configLoader.environment or
```
Comment (emersonloureiro) [inline src/torch/0-tiefvision-commons/tiefvision_config_loader.lua]: Ah, ok this answers my question from above I guess. If so, feel free to ignore it.

File: src/torch/0-tiefvision-commons/tiefvision_torch_io.lua [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  return tiefvision_commons.dataPath(fileName)
+end
+
+function tiefvision_torch_io.read(fileName)
+  local file = filePath(fileName)
+  if not paths.filep(file) then
+    return nil
+  end
+
+  return torch.load(file)
```
Comment (paucarre) [inline src/torch/0-tiefvision-commons/tiefvision_torch_io.lua]: is there any need to close `file`?

Comment (Plippe) [inline src/torch/0-tiefvision-commons/tiefvision_torch_io.lua]: `file` is just a string, so can't really close anything
`torch.load` is identical to your code. No new issue should be injected with this

**Pull request merged**